src/StackExchange.Exceptional.AspNetCore/ConfigSettings.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using StackExchange.Exceptional.Internal;
+using StackExchange.Exceptional.Notifiers;
+using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text.RegularExpressions;
+
+namespace StackExchange.Exceptional
+{
+    /// <summary>
+    /// The Settings element for Exceptional's configuration.
+    /// This is the legacy web.config settings, that only serve as an adapter to populate <see cref="Settings"/>.
+    /// </summary>
+    public partial class ConfigSettings
+    {
+        const string CONFIGSECTION_KEY = "Exceptional";
```

Comment:
Moved the initialization of the Configuration to bind outside our lib in f0aa05a